Add release-4.15 builds

LorbusChris opened the pull request:
Also remove okd-coreos-build pipeline

The okd-coreos-all pipeline can now be configured via parameters, letting us use that for development builds without having to maintain a separate pipeline definition.

Tested locally

LorbusChris committed:
Add release-4.15 builds

Also remove okd-coreos-build pipeline

The okd-coreos-all pipeline can now be configured via parameters,
letting us use that for development builds without having to maintain a
separate pipeline definition.

jcapiitao commented on environments/kind/pipelineruns/okd-coreos-4.14-dev-pipelinerun.yaml:
IIUC `release-stream` value should be `stable`

joelcapitao commented on environments/kind/pipelineruns/okd-coreos-4.14-dev-pipelinerun.yaml:
IIUC `release-stream` value should be `stable`

LorbusChris replied:
For now, 4.13 is still stable, while 4.14 and 4.15 go into next

LorbusChris replied:
There's probably room for improvement for handling building 2 different `next` versions

joelcapitao commented on base/batch/cronjobs/okd-coreos-all-4.14-create-daily.yaml:
`RELEASE_STREAM` value should be `stable`

joelcapitao reviewed (commented):
Just a few in-line comments, other than that LGTM :+1:

LorbusChris committed:
Add COSA imagestream

LorbusChris committed:
Fix PVs

- point to proper device in /dev/
- add claimRef
- move PV for 4.15 to second worker

LorbusChris committed:
Open up workspace permissions for subsequent tasks

LorbusChris committed:
Mount workspace as /workspace

To avoid confusion

LorbusChris committed:
Use `cosa build container`

LorbusChris committed:
Fix EventListener URL in CronJobs

LorbusChris committed:
Build 4.13 base from release-4.14

release-4.13 has the COPR repos that we don't want.
Since we were building from the master branch before,
this should be fine. For the last couple of release before this
branch is dropped.

LorbusChris committed:
cosa-build-baseos: Add 1h30m0s timeout

LorbusChris committed:
Move 4.14 PV to /dev/sdb

IO perf on /dev/sdc is bad

LorbusChris commented:
Everything is operational. Merging
